spec/components/parameters/reports/message/type.ts

```diff
@@ -0,0 +1,19 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../../utils/ref';
+
+const type: ParameterObject = {
+  name: 'type',
+  in: 'query',
+  required: false,
+  description: 'The type of messages.',
```

Comment:
Faltou o the no messages: The type of the messages.

Reply:
O The no inicío que não era necessário por ser título, mas tudo bem. Mas the no "the messages" é necessário.